.gitignore

```diff
@@ -7,3 +7,5 @@ packages/*/src/**/*.d.ts
 dist/
 
 .env
+
+localazy.keys.*.json
```

Comment:
is this for the API key?

Reply:
yes because the cli doesn't give the possibility to use `.env` file but only json file as input.  
And we don't want to expose or API keys to the public (because there are write authorization)